swift/ql/src/queries/Security/CWE-328/WeakSensitiveDataHashing.qhelp

```diff
@@ -51,12 +51,17 @@
             </li>
         </ul>
 
+        <p>
+            Note that special purpose algorithms exist for message authentication (ensuring that a message comes from a particular sender). These algorithms should be used when appropriate, as they address common vulnerabilities of simple hashing schemes in this context.
```

Comment:
The content in brackets breaks the flows. Suggesting the following. Let me know what you think 😃 
```suggestion
            Note that special purpose algorithms, which are usually used to ensure that a message comes from a particular sender, exist for message authentication. These algorithms should be used when appropriate, as they address common vulnerabilities of simple hashing schemes in this context.
```

Reply:
I'm accepting the change but I think I'm going to delete the word "usually".  I think this is _always_ what message authentication algorithms are used for, and even if there are edge cases that isn't the point of this sentence.